Gruntfile.js

```diff
@@ -163,62 +210,30 @@ module.exports = function( grunt ) {
 						]
 					}
 				],
-				files: [
-					"test/data/jquery-1.9.1.js",
-					"external/sinon/sinon.js",
-					"external/npo/npo.js",
-					"external/requirejs/require.js",
-					"test/data/testinit.js",
-
-					"test/jquery.js",
-
-					// Replacement for testinit.js#loadTests()
-					"test/data/testrunner.js",
-					"test/unit/basic.js",
-					"test/unit/core.js",
-					"test/unit/callbacks.js",
-					"test/unit/deferred.js",
-					"test/unit/deprecated.js",
-					"test/unit/support.js",
-					"test/unit/data.js",
-					"test/unit/queue.js",
-					"test/unit/attributes.js",
-					"test/unit/event.js",
-					"test/unit/selector.js",
-					"test/unit/traversing.js",
-					"test/unit/manipulation.js",
-					"test/unit/wrap.js",
-					"test/unit/css.js",
-					"test/unit/serialize.js",
-					"test/unit/ajax.js",
-					"test/unit/effects.js",
-					"test/unit/offset.js",
-					"test/unit/dimensions.js",
-					"test/unit/animation.js",
-					"test/unit/tween.js",
-					"test/unit/ready.js",
-
-					{ pattern: "dist/jquery.*", included: false, served: true },
-					{ pattern: "src/**", included: false, served: true },
-					{ pattern: "external/**", included: false, served: true },
-					{
-						pattern: "test/**/*.@(js|css|jpg|html|xml|svg)",
-						included: false,
-						served: true
-					}
-				],
+				files: getKarmaFiles(),
 				reporters: [ "dots" ],
 				autoWatch: false,
 				concurrency: 3,
 				captureTimeout: 20 * 1000,
 				singleRun: true
 			},
+
 			main: {
 
 				// The Chrome sandbox doesn't work on Travis.
 				browsers: [ isTravis ? "ChromeHeadlessNoSandbox" : "ChromeHeadless" ]
 			},
 
+			mainWithDocumentMode: {
```

Comment:
Do you think this mode is worth it or that we should just assume IE is not going to change too much?

Reply:
This has been removed now.